dev/com.ibm.ws.microprofile.faulttolerance.cdi/src/com/ibm/ws/microprofile/faulttolerance/cdi/config/impl/FallbackConfigImpl.java

```diff
@@ -0,0 +1,211 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
```

Comment:
I think this file was just moved? If so it should still have the original copyright date. Should be `2017, 2019`.